src/ILCompiler.Compiler/src/IL/HelperExtensions.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
It should be fixed be fixed by adding the SystemModule property as we have discussed on the hallway ... we can add the file here now, and move it later ... either way is fine.